central/views/imagecve/view_impl.go

```diff
@@ -0,0 +1,58 @@
+package imagecve
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/walker"
+	"github.com/stackrox/rox/pkg/search"
+	pgSearch "github.com/stackrox/rox/pkg/search/postgres"
+)
+
+type imageCVECoreViewImpl struct {
+	schema *walker.Schema
+	db     *postgres.DB
+}
+
+func (v *imageCVECoreViewImpl) Get(ctx context.Context, q *v1.Query) ([]CveCore, error) {
+	// We only support a dynamic where clause. CveCore has a pre-defined select and group by. Remember this is a "view".
+	if len(q.GetSelects()) > 0 {
```

Comment:
Do we need to worry if select and groupBy are present in a nested query inside the given query?

Reply:
No, we do not have to. The contract is clear.